src/main/java/bussiness_layer/dto/CourseDTO.java

```diff
@@ -31,10 +31,13 @@
     @Min(7)
     private Byte nrOfLaboratories;
 
+    private ProfessorDTO coordinator;
+
     public CourseDTO(Course entity){
         this.id = entity.getId();
```

Comment:
This method should appear in a CourseDtoMapper class, not here.

Reply:
ok